docs/config/failOnZeroTests.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+title: QUnit.config.failOnZeroTests
+excerpt: Fail the test run if no tests were run.
+categories:
+  - config
+version_added: "2.16.0"
+---
+
+Whether to fail the test run if no tests were run.
+
+### Description
+
+<table>
+<tr>
+  <th>type</th>
+  <td markdown="span">`boolean`</td>
+</tr>
+<tr>
+  <th>default</th>
+  <td markdown="span">`true`</td>
+</tr>
+</table>
+
+By default, it is considered an error if no tests were loaded, or if no tests matched the current filter. Turning this option off means an empty test run will result in a success instead.
```

Comment:
"Turning this option off" seems a little odd; maybe "Setting this option to `true`"...

Reply:
Agreed. Done!